### PR TITLE
Added people.deleteUser()

### PR DESCRIPTION
**Pull request by julienroubieu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,11 +51,11 @@ older versions:
 - setOnce(peopleProperties, onSuccess, onFail)
 - trackCharge(amount, chargeProperties, onSuccess, onFail)
 - unset(propertiesArray, onSuccess, onFail)
+- deleteUser()
```
Comment (samzilverberg) [inline README.md]: should probably be: `deleteUser(onSuccess, onFail)`

File: www/mixpanel.js [hunk 2]
```diff
@@ -147,6 +147,10 @@ mixpanel.people.unset = function(propertiesArray, onSuccess, onFail) {
   exec(onSuccess, onFail, 'Mixpanel', 'people_unset', [propertiesArray]);
 };
 
+mixpanel.people.deleteUser = function(propertiesArray, onSuccess, onFail) {
```
Comment (samzilverberg) [inline www/mixpanel.js]: propertiesArray not needed, so just: `function(onSuccess, onFail)`

**Comment by julienroubieu** [comment]
Absolutely! I pushed the fixes. Thank YOU for this plugin :)

**Comment by samzilverberg** [comment]
thanks, will try to push a new version to npm this weekend, maybe even sooner if i get the time

**Pull request merged**